src/XRP/xrp-signer.ts

```diff
@@ -0,0 +1,25 @@
+import { Signer } from '../generated/web/org/xrpl/rpc/v1/transaction_pb'
+
+/*
+ * Represents a signer of a transaction on the XRP Ledger.
+ * @see: https://xrpl.org/transaction-common-fields.html#signers-field
+ */
+export default class XRPSigner {
+  public static from(signer: Signer): XRPSigner | undefined {
+    const account = signer
+      .getAccount()
+      ?.getValue()
```

Comment:
really small nitpick: can you make the ? go on getAccount()'s line ie getAccount()? since that is the thing it belongs to.

Reply:
that is a good nit, but it actually produces a parsing error when I write it that way, and the linter is automatically forcing me to write it as is.